example/bench/bench.cpp

```diff
@@ -11,31 +11,31 @@
 #include <arbor/profile/meter_manager.hpp>
 #include <arbor/common_types.hpp>
 #include <arbor/distributed_context.hpp>
+#include <arbor/domain_decomposition.hpp>
+#include <arbor/load_balance.hpp>
 #include <arbor/profile/profiler.hpp>
 #include <arbor/recipe.hpp>
 #include <arbor/simulation.hpp>
 
-#include "hardware/node_info.hpp"
-#include "load_balance.hpp"
-#include "util/ioutil.hpp"
-
-#include "json_meter.hpp"
+#include <aux/ioutil.hpp>
+#include <aux/json_meter.hpp>
+#include <aux/with_mpi.hpp>
 
 #include "parameters.hpp"
 #include "recipe.hpp"
 
-using namespace arb;
+namespace profile = arb::profile;
 
 int main(int argc, char** argv) {
     try {
-        distributed_context context;
-        #ifdef ARB_HAVE_MPI
-        mpi::scoped_guard guard(&argc, &argv);
+        arb::distributed_context context;
+#ifdef ARB_HAVE_MPI
```

Comment:
We argued about this very recently. And I am certain that we decided to indent preprocessors.

Reply:
But these aren't inside a preprocessor scope.

Reply:
But But, you also indented those (see the code that determines the processor affinity.
I find the "just left indent regardless of the scope in which it resides" makes the code harder to read. Just like not indenting `case` statements ;)